test/test_grocy.py

```diff
@@ -220,6 +218,28 @@ def test_consume_product_error(self):
         )
         self.assertRaises(GrocyError, self.grocy.consume_product, 1, 1.3)
 
+    @pytest.mark.vcr
+    def test_inventory_product_valid(self):
+        currentInv = int(self.grocy.product(4).available_amount)
```

Comment:
nit: variable (and function) names are written in lowercase_with_underscores (https://www.python.org/dev/peps/pep-0008/#id43)

Reply:
Sorted - I am primarily a Javascript man which unfortunately leads to me dropping the ball on occasion